GUIDE_TO_KOGITO_DEVS.md

```diff
@@ -0,0 +1,168 @@
+**Note**: You can check the location of the artifact in the image from the [kogito-images](https://github.com/kiegroup/kogito-images) repository.
+
+Now just build your image. Inside the above created directory run:
+
+`$ podman build -t <prefered-tag-name>  -f jobs-service-Dockerfile  .`
+
+The above command will build your image with `<prefered-tag-name>` and can be seen in the output of `$ podman images`
+
+**Note**: We added the `-f` flag to specify the `Dockerfile` in case you want to build multiple images, you can then have different files and specify them with the `-f` flag.
+This image needs to be on a public container registry from which it can be pulled later.
+```shell-script
+$ podman tag <prefered-tag-name> quay.io/<your-namespace>/<image-name>
+Eg: podman tag jobs-service quay.io/myorg/jobs-service:latest
+$ podman push quay.io/<your-namespace>/<image-name>
+```
+
+## Build the Kogito Service Image
+
+ ### Base Images
+  * kogito-quarkus-ubi8
+  * kogito-quarkus-jvm-ubi8
+  * kogito-springboot-ubi8
+
+As an example for building the Image for your Kogito custom Service we’re going to build the example  `process-business-rules-quarkus` from [kogito-examples](https://github.com/kiegroup/kogito-examples) repository.
+
+```shell-script
+cd kogito-examples/process-business-rules-quarkus
+mvn clean package
+```
+Now create a Dockerfile inside the `kogito-examples/process-business-rules-quarkus`. With the following content:
+
+```Dockerfile
+FROM quay.io/kiegroup/kogito-quarkus-jvm-ubi8:latest
+
+COPY target/*-runner.jar $KOGITO_HOME/bin    
+COPY target/lib $KOGITO_HOME/bin/lib
+```
+
+```shell-script
+$ podman build --tag quay.io/<yournamespace>/process-business-rules-quarkus:latest .
+```
+It’ll build your image with the kogito service running on it.
+
+Now let’s push the above built image to a public container registry.
+
+`$ podman push quay.io/<yournamespace>/process-business-rules-quarkus:latest`
+
+## Install the Operator
+First we would need to enable the OLM in our Minikube cluster, for that just run:
+
+`$ minikube addons enable olm`
+
+To launch the OLM console locally, clone the [operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager) repository and from the root of the project run: `$ make run-console-local`.
+ 
+This will run the operatorhub console on http://localhost:9000 
+
+**Note**: You will need to have [`jq`](https://stedolan.github.io/jq/manual/) installed and 9000 port available on the system.
+
+Create a different namespace where `kogito-operator` and all the dependent operator will run
+
+`$ kubectl create ns kogito`
+
+Now open your browser and visit the OLM console on https://localhost:9000. Select `Operators > OperatorHub` and search for `kogito`. 
+Select the Kogito Operator by Red Hat and install it with defaults only changing the namespace where it needs to be installed. You can select the `kogito` namespace which was created for this purpose.
```

Comment:
`and install it with defaults only changing the namespace where it needs to be installed`
can you rephrase ?